src/lib/rest/httpRequestSend.cpp

```diff
@@ -131,6 +131,42 @@ static char* curlVersionGet(char* buf, int bufLen)
   return buf;
 }
 
+
+
+/* ****************************************************************************
+*
+* httpHeaderAdd - 
+*/
```

Comment:
Sure.
Actually, that is ALWAYS a good idea. 
Perhaps we should add it to the style guide ...